### DISCUSSION TITLE
Multilple Bids for different assets?

### DISCUSSION THREAD

**Discussion by jack75015** [ProjectOpenSea/seaport-js#67] [Q&A]
Hey 

I managed to make via seaportJS a bid on an ERC721 asset (1 <-> 1) and I was wondering if it was possible to make several bids on different assets?

for example:

```
{
   0.5 WrappedETH -> 1 NFT from contract 0x123..
   0.4 WrappedETH -> 1 NFT from contract 0x456..
   0.6 WrappedETH -> 1 NFT from contract 0x789..
}
```

If yes, does someone have examples on it?

Thanks :)

**Comment by ryanio** [comment]
for individual offers of differing amounts for different contract items I believe you'll need to create separate orders. note that these separate orders can be executed together to improve gas efficiency.

if you would like to make the same offer on multiple items in a single collection, you can use `criteria` (read more about it in the [docs](https://github.com/ProjectOpenSea/seaport/blob/main/docs/SeaportDocumentation.md)). an advanced order with partial fill would allow the same order to be fulfilled multiple times rather than just once.
